exercises/005_create_transfer.sh

```diff
@@ -19,4 +19,4 @@ tb "create_transfers debit_account_id=500 credit_account_id=501 amount=100 ledge
 # But we wouldn't recommend doing this in production!
 
 # We recommend using TigerBeetle Time-Based Identifiers for most IDs:
-# https://docs.tigerbeetle.com/coding/data-modeling#id
```

Comment:
This section was renamed but we hadn't deployed the latest changes to the live docs site yet. It's being deployed now

Reply:
Had not noticed it. Reverted